FhirToDataLake/deploy/templates/ContainerApp/DeployFhirPipelineToContainerApp.json

```diff
@@ -1,420 +1,447 @@
+          }
+      },
+      "fhirServerUrl": {
+          "type": "string",
+          "metadata": {
+              "description": "The fhir server endpoint you wish to get data."
+          }
+      },
+      "authentication": {
+          "type": "bool",
+          "defaultValue": false,
+          "allowedValues": [
+          true,
+          false
           ],
-          "scale": {
-            "minReplicas": 1,
-            "maxReplicas": "[parameters('maxInstanceCount')]",
-            "rules": [
-                {
-                "name": "cpu-scaling-rule",
-                "custom": {
-                    "type": "cpu",
-                    "metadata": {
-                        "type": "Utilization",
-                        "value": "50"
-                        }
-                    }
-                },
-                {
-                    "name": "memory-scaling-rule",
-                    "custom": {
-                        "type": "memory",
-                        "metadata": {
-                            "type": "Utilization",
-                            "value": "50"
-                        }
-                    }
-                }
-            ]
+          "metadata": {
+              "description": "Whether to access Azure API for FHIR with managed identity authentication. Set false if you are using an OSS public FHIR server instance."
+          }
+      },
+      "fhirVersion": {
+          "type": "string",
+          "defaultValue": "R4",
+          "allowedValues": ["R4"]
+      },
+      "dataStart": {
+          "type": "string",
+          "defaultValue": "1970-01-01 00:00:00 +00:00",
+          "metadata": {
+              "description": "Start timestamp of the data range you want to export."
+          }
+      },
+      "dataEnd": {
+          "type": "string",
+          "defaultValue": "",
+          "metadata": {
+              "description": "End timestamp of the data range you want to export. Will continuous export all data if not specified."
+          }
+      },
+      "containerName": {
+          "type": "string",
+          "defaultValue": "fhir",
+          "metadata": {
+              "description": "The name of the container to store job and data."
+          }
+      },
+      "jobConcurrency": {
+          "type": "int",
+          "defaultValue": 3,
+          "minValue": 2,
+          "maxValue": 10,
+          "metadata": {
+              "description": "Concurrent jobs executing in parallel."
+          }
+      },
+      "schedulerIntervalInMinutes": {
+          "type": "int",
+          "defaultValue": 5,
+          "allowedValues": [
+          0,
+          5,
+          30,
+          60,
+          1440
+          ],
+          "metadata": {
+              "description": "The scheduler interval in minutes, schedulerCronExpression will take effect if set to 0."
+          }
+      },
+      "schedulerCronExpression": {
```

Comment:
What if **schedulerCronExpression** is invalid? Will the pipeline throw exception when initializing?

Reply:
yes，if schedulerCronExpression is invalid, the pipeline will throw ConfigurationErrorException at the configuration validator, and provision will fail